deploy/complete/helm-chart/provision/values.yaml

```diff
@@ -8,6 +8,8 @@ global:
     compartmentId:
     # Service instance name
     instanceName: mushop
+    # OSB Namespace
+    namespace: mushop-utilities
```

Comment:
Don't think we should be specifying this here, because the user is instructed to install with helm `--namespace mushop`

Reply:
after discussion this should be removed and clusterservicebroker can be referenced without namespace path

Reply:
Need to install with --namespace mushop-utilities, to be separated from the mushop micro services. That entry is not needed if fully use the osb as ClusterServiceBroker

Reply:
agree to remove

Reply:
New commit removed namespaces service broker